outputs.tf

```diff
@@ -1,5 +1,20 @@
+
+output "protected_nat_instance_eips" {
+  description = "List of Elastic IP addresses used by the NAT instances with `prevent_destroy` set. This will be empty if EIPs are provided in var.nat_instance_eip_ids."
+  value       = local.reuse_nat_instance_eips ? [] : aws_eip.nat_instance_eips[*].public_ip
+}
+
+output "protected_nat_gateway_eips" {
+  description = "List of Elastic IP addresses used by the standby NAT gateways."
+  value = [
+    for eip in aws_eip.nat_gateway_eips
+    : eip.public_ip
+    if var.create_nat_gateways
+  ]
+}
```

Comment:
Rather than duplicating these outputs, can we use a conditional on the existing outputs? Or create a `local` with the correct outputs, and use that?

Reply:
I pushed a modification in that way, hoping it will fit you.